ui/app/styles/components/tabs.scss

```diff
@@ -13,20 +13,26 @@
       &:focus {
         box-shadow: none;
       }
-      &.is-active,
-      &.is-active .tab {
+      &.active,
```

Comment:
This looks good. I wonder why the `activeClass` override was there in the first place? Was there an alternate styling for `.active` that was being applied maybe?

Reply:
Honestly, no idea. But good idea. Let me do some more checking on .active anywhere else.